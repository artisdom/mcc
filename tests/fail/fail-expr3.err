Type error: expected one of [float] but got int. Error occured in statement:
b + a;