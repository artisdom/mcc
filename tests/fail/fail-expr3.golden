Type error: expected one of [ void *
                            , int
                            , float ] but got float. Error occured in statement:
b + a;